inference-engine/thirdparty/clDNN/src/loop.cpp

```diff
@@ -134,7 +134,7 @@ layout loop_inst::calc_output_layout(loop_node const & node) {
     if (axis_to_iterate_throgh != -1) {
         const auto shape = loop_output_layout.size.sizes(loop_output_layout.format);
         const size_t ndim = shape.size();
-        const size_t raw_axis = node.convert_to_raw_axis(axis_to_iterate_throgh, ndim);
+        const size_t raw_axis = node.convert_to_raw_axis(axis_to_iterate_throgh, (int)ndim);
```

Comment:
this c-style casting is not acceptable. please use c++ style casting

Reply:
Fixed it